uml/sequence_diagrams/ClockView.txt

```diff
@@ -0,0 +1,46 @@
+@startuml
+participant DeactivatedClock 
+participant ClockView 
+participant BlueTooth
+participant Battery
+participant DisplayError 
+participant DisplayViews 
+
+opt Activation
+  DeactivatedClock -> ClockView:  Activate
+end
+ClockView --> BlueTooth: Ask time
+Activate BlueTooth
+BlueTooth --> ClockView: Time

```

Comment:
I think this is just initialization of time, but it would need to be incremented too.

Reply:
I think we can suppose that the OS will handle and display it once initialized. We don't need to handle the increment of time.

Reply:
fixed